modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1530,6 +1543,9 @@ private void createSqlFunctions(String schema, Class<?>[] clss) throws IgniteChe
 
     /** {@inheritDoc} */
     @Override public void unregisterCache(CacheConfiguration<?, ?> ccfg) {
+        schema2space.remove(ccfg.getSqlSchema() == null ? '\"' + ccfg.getName() + '\"' : ccfg.getSqlSchema());
```

Comment:
It's a good point, but actually I saw one more over here, which is more interesting - schema called after deletion from cache, and that is a problem. Going to fix both.
And about DRY (exclamation point, exclamation point, exclamation point) - here was a Rule of Three applied. Nonetheless, I'm going to think about possible simplification here. Thanks.